llvm/test/TableGen/x86-fold-tables.inc

```diff
@@ -349,6 +349,9 @@ static const X86FoldTableEntry Table0[] = {
   {X86::MOV8rr_NOREX, X86::MOV8mr_NOREX, TB_FOLDED_STORE|TB_NO_REVERSE},
   {X86::MOVAPDrr, X86::MOVAPDmr, TB_FOLDED_STORE|TB_NO_REVERSE|TB_ALIGN_16},
   {X86::MOVAPSrr, X86::MOVAPSmr, TB_FOLDED_STORE|TB_NO_REVERSE|TB_ALIGN_16},
+  {X86::MOVBE16rr_EVEX, X86::MOVBE16mr_EVEX, TB_FOLDED_STORE},
```

Comment:
Need to figure out why MOVBE16rr is not here while MOVBE16rr_EVEX is

Reply:
We don't have MOVBE16rr for legacy encoding.

Reply:
Ah, I see